### PR TITLE
Add Tragicforce 68

### PR DESCRIPTION
**Pull request by nooges**

### CODE REVIEW

File: keyboards/tragicforce68/tragicforce68.h [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    { D1, D2, D3, D4, D5, D6, D7, D8 }, \
+    { E1, E2, E3, E4, E5, E6, E7, E8 }, \
+    { F1, F2, F3, F4, F5, F6, F7, F8 }, \
+    { G1, G2, G3, G4, G5, G6, G7, G8 }, \
+    { H1, H2, H3, H4, H5, H6, KC_NO, KC_NO }, \
+    { J1, J2, J3, J4, J5, J6, J7, J8 } \
+}
+
+#define LAYOUT_kc( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K10, K11, K12, K13, K14,     K15, K16, \
```
Comment (mechmerlin) [inline keyboards/tragicforce68/tragicforce68.h]: Why the difference between using `A1, A2, B1, B2` vs the standard `K01, K02 etc etc`?

Comment (nooges) [reply keyboards/tragicforce68/tragicforce68.h]: The `A1, A2, ...` stuff indicates the physical positioning of the wiring, whereas the latter was a cut/paste job from the MF68 port I had done.

File: keyboards/tragicforce68/rules.mk [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+
+# Boot Section Size in *bytes*
+#   Teensy halfKay   512
+#   Teensy++ halfKay 1024
+#   Atmel DFU loader 4096
+#   LUFA bootloader  4096
+#   USBaspLoader     2048
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```
Comment (drashna) [inline keyboards/tragicforce68/rules.mk]: This isn't technically needed, since the bootloader is defined.

File: keyboards/tragicforce68/rules.mk [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE ?= no      # Virtual DIP switch configuration(+1000)
```
Comment (drashna) [inline keyboards/tragicforce68/rules.mk]: Could you remove the question marks from all of these?

**Comment by drashna** [comment]
Thanks! 

And love the name! :D

**Pull request merged**